pkgs/applications/networking/instant-messengers/caprine-bin/build-from-dmg.nix

```
@@ -0,0 +1,38 @@
+{ stdenvNoCC
+, lib
+, fetchurl
+, undmg
+}:
+
+{ pname
+, version
+, sha256 ? lib.fakeSha256
```

```suggestion
, sha256
```
sha256 should always be supplied and can be overwritten for testing purposes to lib.fakeSha256.